webpack/components/extensions/RegistrationCommands/__tests__/__snapshots__/ActivationKeys.test.js.snap

```diff
@@ -2,7 +2,7 @@
 
 exports[`ActivationKeys renders 1`] = `
 <FormGroup
-  fieldId="activation_keys_field"
+  fieldId="reg_katello_ak"
```

Comment:
I thought QE wanted us to remove "katello" from these? Or did I have that switched around?